tests/workload/nvidia-smoke/README.md

```diff
@@ -0,0 +1,90 @@
+# NVIDIA GPU Smoke tests
+
+The image created by the Dockerfile in this folder compiles a few [CUDA samples](https://github.com/NVIDIA/cuda-samples/) for both `x86_64` and `aarch64`.
+This container image can be used as a TestSys workload test to validate the Bottlerocket `*-nvidia` variants running on NVIDIA hosts.
+
+## Build requirements
+
+Due to missing packages needed for cross-compiling, these images need to be created on an **arm64** host.
+
+In order to build the image for both architectures, you need to set up the `buildx` docker plugin.
+[This guide describes](https://github.com/docker/buildx#installing) the installation process.
+
+By default, the install of `docker buildx` uses the `docker` driver type.
+This driver is not able to build for all platform targets we need.
+You will need to run the following to create a new builder using the `docker-container` driver:
+
+```bash
+docker buildx create --use --bootstrap
+docker buildx ls
+```
+
+You should see a `*` next to your newly created builder denoting it as the currently active builder to use.
+If it is not, you can switch builder contexts by running:
+
+```sh
+docker buildx use $BUILDER_NAME
+```
+
+Supported platforms can be verified by running:
+
+```bash
+docker buildx inspect | grep Platforms
+```
+
+The output from this command should show both `linux/amd64` and `linux/arm64` platforms.
+Compiling the `amd64` binaries on an `arm64` host using emulation will be significantly slower.
+See below for instructions on how to set up a remote build host to offload this activity.
+
+This guide assumes you use ECR as your container's registry, and that you already have the ECR credentials plugin configured in your host.
```

Comment:
Is it possible to write this guide without assuming `ECR`

Reply:
Reworded this note since there really isn't anything that assumes or expects ECR vs any other image registry.